src/SendGrid/SendGrid.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <VersionPrefix>9.9.0</VersionPrefix>
-    <TargetFrameworks>netstandard1.3;net452</TargetFrameworks>
+    <TargetFrameworks>netstandard1.3;netstandard2.0;net452</TargetFrameworks>
```

Comment:
There is no point in adding both `netstandard1.3` and `netstandard2.0` as targets unless you have conditionally-compiled code that leverages APIs not available in `netstandard1.3`.